keanu/routes/orders.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+import json
+from flask import Blueprint, jsonify, request
+from flask_autodoc import Autodoc
+from keanu.routes.login import login_api
+
+order_api = Blueprint('orderApi', __name__)
+
+auto = Autodoc()
+
+
+@order_api.route('/order/spec')
+def order_doc():
+    """
+    Documentation for the /order route
+    :return:
+    """
+    return auto.html()
+
+
+@order_api.route('/order', methods=['GET'])
+@auto.doc()
+def get_user_orders() -> dict:
+    """
+    returns all the orders for user as a json array
+    :return:
+    """
+    from keanu.models.orders import Order
+    #from keanu.models.users import User
+
+    # get all orders
+    if ('_Id' in request.headers) :
+        id = request.headers['_id']
+        orders = Order.query.filter(id)
+        # create orders list
+        orders_list = []
+        # create response
+        for order in orders:
+            orders_list.append({
+                "_id": str(order.mongo_id),
+                "items": order.items,
+                "total": order.total,
+                "userId": order.userId,
```

Comment:
We need to find a way to add get the userId the request somehow because the frontend wouldn't have the userId